### PR TITLE
Added travis file from symfony best practices

### PR DESCRIPTION
**Pull request by Nyholm**
This will also test dunglas/symfony-lock

### CODE REVIEW

**Review by dunglas — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -21,35 +16,44 @@ branches:
 matrix:
     fast_finish: true
     include:
-        # Test stable versions
+          # Test with lowest dependencies
+        - php: 7.1
```
Comment (dbu) [inline .travis.yml:20]: imho this should also be the oldest php version we support to really get the oldest possible thing

**Review by dbu — COMMENTED** [review]
apart from the php version for --prefer-lowest i am happy with this

**Review by dbu — APPROVED** [review]

**Comment by dbu** [comment]
are you happy with this as is, or do you want to wait for further input on the symfony-docs pull request?

**Review by fbourigault — APPROVED** [review]

**Comment by Nyholm** [comment]
I’m very happy :)

**Pull request merged**